examples/12_css/vite.config.ts

```diff
@@ -1,6 +1,10 @@
 import { defineConfig } from 'vite';
 import { vanillaExtractPlugin } from '@vanilla-extract/vite-plugin';
+import styleX from 'vite-plugin-stylex';
 
 export default defineConfig({
-  plugins: [vanillaExtractPlugin({ emitCssInSsr: true })],
+  ssr: {
+    external: ['@stylexjs/stylex'],
+  },
```

Comment:
Does it work as expected?
Anyway, I would like to merge #289 first. Please be patient.

Reply:
yes